application/src/main/kotlin/net/corda/v5/application/crypto/SigningService.kt

```diff
@@ -35,4 +35,15 @@ interface SigningService {
      */
     @Suspendable
     fun sign(bytes: ByteArray, publicKey: PublicKey, signatureSpec: SignatureSpec): DigitalSignature.WithKey
+
+    /**
+    * TODO Remove this API from C5. Added as a temporary API for testing until MGM code is fully integrated.
```

Comment:
I wonder if it's worth having this API anyway to make the process of passing a public key in from RPC better? It's probably a question that needs wider discussion with @rick-r3 and @alexey-kadyrov-r3, as it depends quite a lot on what we're expecting users to do with crypto.